docs/Cloud_integrations/LoRaWAN/LoRagateway.md

```diff
@@ -0,0 +1,101 @@
+        - *Gateway ID*:  A unique identifier for your gateway.
+    - *No gateway EUI* - select this option if your gateway doesn't have a Gateway EUI. Provide the information in one field:
+        - *Endpoint name - gateway ID*: A unique identifier for your gateway. It will become the endpoint name by which the platform server will identify the device.
+- *Friendly name* - Gateway name: An optional human-readable name to help you identify your gateway. The Things Stack will use this name instead of the gateway ID to refer to your gateway.
+
+In the section *Settings*, select your Frequency plan. The frequency plan defines data rates that your end device or gateway is set up to use. The gateway and end devices within reach must use the same frequency plan to communicate. 
+
+In the section *LwM2M security mode for device-server interaction* select the security mode:
+
+- Pre-shared key - authorize your device using a pre-shared key. To use this option, provide the following:
+    - Key identity - define the name that the device will use during the DTLS handshake. Can be equal to the LwM2M device endpoint name.
+    - Key format. 
+    - Key - provide the shared secret used in the device-server authentication.
+- NoSec - use the device with no security established for the device-server communication. Use this mode only for testing purposes.
+- Certificate - secure your device-server communication with a certificate:
+    - Select *Use a previously uploaded certificate* if you have already uploaded a certificate to the platform using the DTLS/TLS Certificates panel.
+    - Select *Upload a new certificate* to load your certificate by clicking **Browse**.
+
+![Step 1](images/Step_1.png)
+
+### Configure server connection 
+
+This step is only for the connection via the Bootstrap Server. 
+
+
+Provide new bootstrap connection settings. You can choose from the options in the drop-down list:
+
+- *Use the bootstrap configuration set as default for your domain* - use this setting if you want to employ the inherited bootstrap settings for your domain.
+- *This Coiote IoT DM Management Server (Cert with EST)* - use this setting if you have chosen the Certificate mode in the Device credentials step.
+- *This Coiote IoT DM Management Server NoSec* - use this setting to establish the device-server communication with no security. Using this mode is not recommended except for testing purposes.
+- *This Coiote IoT DM Management Server PSK* - use this setting to authorize the device-server communication using a pre-shared key.
+- *Your custom bootstrap setting* - this option is available if you configured a bootstrap server setting using the Bootstrap configuration panel.
+
+![Step 2](images/Step_2.png)
+
+### Connect your device (optional)
+
+In the last step, you can connect your device by inserting the displayed parameters into your device. It's an optional step. You can finish the configuration and connect the device later.
+
+![Step 3](images/Step_3.png)
```

Comment:
The `edfgfd` as `Key identity` and `cdm-testmaster` as `URL` looks a bit... strange. Can you please change it to something more "official"?

Reply:
Shouldn't the URL be the Coiote's URL instead of LoRaWAN Network's one? That looks like pure LwM2M configuration.

Reply:
I would be grateful if you could provide the example URL that you think should be visible on this screen.

Reply:
I believe the most basic Coiote URL will be enough: `coaps://eu.iot.avsystem.cloud:5684`.